balancer/rls/balancer.go

```diff
@@ -65,7 +65,8 @@ var (
 	resetBackoffHook     = func() {}
 )
 
-const balancerName = "rls_experimental"
+// Name is the name of the RLS LB policy.
+const Name = "rls_experimental"
```

Comment:
Can you update https://github.com/grpc/grpc-go/blob/eb6ff1ae43acd46fe708dee34997317a79d5136f/xds/internal/clusterspecifier/rls/rls.go#L39 along with this?

Reply:
Done. Thanks.